packages/pcd/rln-pcd/src/RLNPCD.ts

```diff
@@ -20,7 +20,7 @@ import { Identity } from "@semaphore-protocol/identity";
 import JSONBig from "json-bigint";
 import { Proof, RLN, RLNFullProof } from "rlnjs";
 import { v4 as uuid } from "uuid";
-import verificationKeyJSON from "../artifacts/16.json";
+import verificationKeyJSON from "./16.json";
```

Comment:
Interesting.  Why did this file need to move when the artifacts of other ZK PCDs didn't?

Reply:
For TypeScript's language server to be able to generate its own in-memory representation of the full type structure, we need `tsconfig.json` to accurately reflect the structure of the code base. This means that `rootDir` needs to be the actual source root, and files cannot be included from outside of it. The JSON file is the only artifact file that gets included by TypeScript, and so has to move to somewhere inside of the source root, or TypeScript will fail to generate types.

Reply:
Looking deeper, it seems the answer is that the other ZK PCDs did in fact need this change, and I found the point where the file is moved and the artifact generation script is updated.  I missed that last night, possibly because I was looking at the `-ui` package instead of the main PCD package.  Sorry for the confusion, I understand the reason for this change.